README.md

```diff
@@ -1,6 +1,35 @@
 # Nukta
 > الجزايريين زهوانيين يحبو يضحكو
 
+## Getting started
+Install:  
+`yarn add nukta.dz` or `npm i --save nukta.dz`
+
+Start using:  
+```js
+const nukta = require('nukta.dz')
+
+console.log(`We have ${nukta.count} jokes available for you! ashba3 da7k ;P`)
+
+// Set all jokes
+console.log(nukta.nukat)
```

Comment:
I think both are interesting and have benefits.
I may want to get the whole list and alter it's form or expose it in a different form to another API ...etc.

So I'll do both 😉